package.json

```diff
@@ -50,6 +50,7 @@
     "webpack-node-externals": "^1.5.4"
   },
   "dependencies": {
-    "hpq": "^1.1.1"
+    "hpq": "^1.1.1",
+    "lodash": "^4.17.4"
```

Comment:
Could we add some technical guidelines to the docs of this project to explain some choices? React, JSX, how we use lodash, ... 

I'm ok not using `lodash` for now. ES6 features are not as "complete" as lodash, but we can do a lot without any lib.